includes/admin/settings/class-wcmp-settings-data.php

```diff
@@ -670,7 +670,7 @@ private function get_section_export_defaults_main()
                 "name"      => WCMYPA_Settings::SETTING_LABEL_DESCRIPTION,
                 "label"     => __("Label description", "woocommerce-myparcel"),
                 "help_text" => __(
-                    "With this option you can add a description to the shipment. This will be printed on the top left of the label, and you can use this to search or sort shipments in your backoffice.",
+                    "With this option you can add a description to the shipment. This will be printed on the top left of the label, and you can use this to search or sort shipments in your backoffice. Because of limited space on the label that varies per package type, we recommend that you keep a label description as short as possible.",
```

Comment:
ik denk "the label description" ipv "a label description"

Reply:
Dit is precies te text die in de story staat, ik zal het met Mitchell bespreken

Reply:
`...space on the label that varies...`

"that" moet "which" zijn. En wat hij zei!